### PR TITLE
Feat: Gallery Section

### PR DESCRIPTION
**Pull request by shafdo**

### CODE REVIEW

File: app/_components/Home/Gallery/index.jsx [hunk 1]
```diff
@@ -1,9 +1,68 @@
...
+        Gallery
+      </h1>
+
+      <div className="gallery space-y-4 py-16 container block mx-auto overflow-x-hidden">
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="left"
+            className="flex"
```
Comment (sathninduk) [inline app/_components/Home/Gallery/index.jsx:18]: `flex` within an element that already has `flex` is redundant.

File: app/_components/Home/Gallery/index.jsx [hunk 2]
```diff
@@ -1,9 +1,68 @@
...
+              />
+            ))}
+          </Marquee>
+        </div>
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="right"
+            className="flex"
```
Comment (sathninduk) [inline app/_components/Home/Gallery/index.jsx:35]: `flex` within an element that already has `flex` is redundant.

File: app/_components/Home/Gallery/index.jsx [hunk 3]
```diff
@@ -1,9 +1,68 @@
...
+            ))}
+          </Marquee>
+        </div>
+
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="left"
+            className="flex"
```
Comment (sathninduk) [inline app/_components/Home/Gallery/index.jsx:53]: `flex` within an element that already has `flex` is redundant.

File: app/_components/Home/Gallery/index.jsx [hunk 4]
```diff
@@ -1,9 +1,68 @@
...
+      <div className="gallery space-y-4 py-16 container block mx-auto overflow-x-hidden">
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="left"
+            className="flex"
+          >
+            {galleryJson.row1.map((image, index) => (
+              <img
```
Comment (sathninduk) [inline app/_components/Home/Gallery/index.jsx:21]: Consider using `Image` from `next/image` to enhance performance (Along with adequate `width` and `height` props).

File: app/_components/Home/Gallery/index.jsx [hunk 5]
```diff
@@ -1,9 +1,68 @@
...
+        </div>
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="right"
+            className="flex"
+          >
+            {galleryJson.row2.map((image, index) => (
+              <img
```
Comment (sathninduk) [inline app/_components/Home/Gallery/index.jsx:38]: Consider using `Image` from `next/image` to enhance performance (Along with adequate `width` and `height` props).

File: app/_components/Home/Gallery/index.jsx [hunk 6]
```diff
@@ -1,9 +1,68 @@
...
+
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="left"
+            className="flex"
+          >
+            {galleryJson.row3.map((image, index) => (
+              <img
```
Comment (sathninduk) [inline app/_components/Home/Gallery/index.jsx:56]: Consider using `Image` from `next/image` to enhance performance (Along with adequate `width` and `height` props).

**Review by sathninduk — CHANGES_REQUESTED** [review]

File: app/_components/Home/Gallery/index.jsx [hunk 7]
```diff
@@ -1,9 +1,68 @@
...
+
+      <div className="gallery space-y-4 py-16 container block mx-auto overflow-x-hidden">
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="left"
+            className="flex"
+          >
+            {galleryJson.row1.map((image, index) => (
```
Comment (sathninduk) [inline app/_components/Home/Gallery/index.jsx:20]: Ensure `galleryJson.row1` is defined and not null before using `map`.

File: app/_components/Home/Gallery/index.jsx [hunk 8]
```diff
@@ -1,9 +1,68 @@
...
+          </Marquee>
+        </div>
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="right"
+            className="flex"
+          >
+            {galleryJson.row2.map((image, index) => (
```
Comment (sathninduk) [inline app/_components/Home/Gallery/index.jsx:37]: Ensure `galleryJson.row2` is defined and not null before using `map`.

File: app/_components/Home/Gallery/index.jsx [hunk 9]
```diff
@@ -1,9 +1,68 @@
...
+        </div>
+
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="left"
+            className="flex"
+          >
+            {galleryJson.row3.map((image, index) => (
```
Comment (sathninduk) [inline app/_components/Home/Gallery/index.jsx:55]: Ensure `galleryJson.row3` is defined and not null before using `map`.

**Review by sathninduk — CHANGES_REQUESTED** [review]

File: app/_components/Home/Gallery/index.jsx (hunk 1 above)
Comment (shafdo) [reply app/_components/Home/Gallery/index.jsx:18]: Resolved

File: app/_components/Home/Gallery/index.jsx (hunk 2 above)
Comment (shafdo) [reply app/_components/Home/Gallery/index.jsx:35]: Resolved

File: app/_components/Home/Gallery/index.jsx (hunk 3 above)
Comment (shafdo) [reply app/_components/Home/Gallery/index.jsx:53]: Resolved

File: app/_components/Home/Gallery/index.jsx (hunk 4 above)
Comment (shafdo) [reply app/_components/Home/Gallery/index.jsx:21]: Resolved

File: app/_components/Home/Gallery/index.jsx (hunk 5 above)
Comment (shafdo) [reply app/_components/Home/Gallery/index.jsx:38]: Resolved

File: app/_components/Home/Gallery/index.jsx (hunk 6 above)
Comment (shafdo) [reply app/_components/Home/Gallery/index.jsx:56]: Resolved

File: app/_components/Home/Gallery/index.jsx (hunk 7 above)
Comment (shafdo) [reply app/_components/Home/Gallery/index.jsx:20]: Resolved

File: app/_components/Home/Gallery/index.jsx (hunk 8 above)
Comment (shafdo) [reply app/_components/Home/Gallery/index.jsx:37]: Resolved

File: app/_components/Home/Gallery/index.jsx (hunk 9 above)
Comment (shafdo) [reply app/_components/Home/Gallery/index.jsx:55]: Resolved

**Review by Randula98 — APPROVED** [review]
LGTM

**Pull request merged**